dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -8,6 +8,64 @@
   }
 end
 
+Given /^I am a CSF facilitator named "([^"]*)" for regional partner "([^"]*)"$/ do |facilitator_name, partner_name|
+  require_rails_env
+
+  RegionalPartner.find_or_create_by(name: partner_name, group: 1)
+
+  email, password = generate_user(facilitator_name)
+  facilitator = FactoryGirl.create(:facilitator, name: facilitator_name, email: email, password: password)
+  FactoryGirl.create(:pd_course_facilitator, course: Pd::Workshop::COURSE_CSF, facilitator: facilitator)
+
+  # CSF Facilitators are also workshop organizers
+  facilitator.permission = UserPermission::WORKSHOP_ORGANIZER
```

Comment:
I thought we deprecated this permission. If this is to enable CSF facilitators to create workshops, this shouldn't be necessary. Should be taken care of here https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/ability.rb#L114

Reply:
This is nice, because now I can reuse the step `Given /^There is a facilitator named "([^"]+)" for course "([^"]+)"$/ do |name, course|`